{{cookiecutter.github_repository_name}}/docs/api/users.md

```diff
@@ -7,57 +7,47 @@ Supports registering, viewing, and updating user accounts.
 
 `POST` `users/`
 
+Parameters:
+
+Name     | Type   | Description
+---------|--------|---
+username | string | The username for the new user.
+password | string | The password for the new user (plaintext).
+
 *Note:*
 
 - *Not* **Authorization Protected**
 
 
 **Response**:
+If a User with given `username` doesn't exist, you'll get a 201:
 
 ```json
 Content-Type application/json
 201 Created
 
 {
-  "id": 1,
-  "first_name": "Richard",
-  "last_name": "Hendriks",
-  "auth_token": "fFBGRNJru1FQd44AzqT3Zg",
-  "email": "hendriks@piepiper.com"
+  "id": "6d5f9bae-a31b-4b7b-82c4-3853eda2b011",
+  "username": "richard",
+  "password": "pbkdf2_sha256$24000$aGozcCr6QXhv$WCgPt2voqVO+Nno2flVnNnLcfks6Yq8XJyxoadB/r50=",
+  "auth_token": "132cf952e0165a274bf99e115ab483671b3d9ff6"
 }
 ```
-## Get a user's profile information
-
-**Request**:
-
-`GET` `users/:id`
-
-Parameters:
-
-Name | Type | Description
----|---|---
-id | integer | The id associated with the user object.
-
-
-*Note:*
 
-- **[Authorization Protected](authentication.md)**
-
-**Response**:
+else if the `username` is already taken, you'll get a 400:
 
 ```json
 Content-Type application/json
-200 OK
+400 Bad Request
```

Comment:
Covered in [comment](https://github.com/agconti/cookiecutter-django-rest/pull/171/files#r53727156).